deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -395,6 +395,12 @@ spec:
           - create
           - update
           - delete
+        - apiGroups:
```

Comment:
You need to ensure that these are added into the role manifests in the Rook repository. With automated generation of CSV, this will get overwritten if this is not persent in Rook.

Reply:
Rook will not take any monitoring changes.

Reply:
https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/monitoring/rbac.yaml